src/core/lib/slice/slice_internal.h

```diff
@@ -309,4 +312,17 @@ inline uint32_t grpc_slice_hash_internal(const grpc_slice& s) {
 // 0. All other slices will return the size of the allocated chars.
 size_t grpc_slice_memory_usage(grpc_slice s);
 
+inline grpc_slice grpc_slice_from_static_buffer_internal(const void* s,
```

Comment:
If we want all internal callers to be using these `_internal()` methods instead of the external ones, let's update tools/run_tests/sanity/core_banned_functions.py to enforce that.

Reply:
Done.